src/main/java/com/booksaw/betterTeams/commands/team/DisbandCommand.java

```diff
@@ -27,33 +27,18 @@ public class DisbandCommand extends TeamSubCommand {
 
 	@Override
 	public CommandResponse onCommand(TeamPlayer teamPlayer, String label, String[] args, Team team) {
+		UUID playerId = teamPlayer.getPlayer().getUniqueId();
 
-		UUID found = null;
-		// if they have already had the confirm dialogue
+		if ((args.length > 1 && args[0].equals("confirm")) ||
```

Comment:
(a bug it seems I introduced a while ago) Am I going crazy or should I have put args.length >= 1 here?

Reply:
Well, to check args[0], you only need 1 argument, so >= 1 should be better yes.
Shall I adjust it?

Reply:
Yeah, please do :)

Reply:
done